pkg/config/user_config.go

```diff
@@ -496,28 +496,28 @@ func GetDefaultConfig() *UserConfig {
 				FetchRemote:            "f",
 			},
 			Commits: KeybindingCommitsConfig{
-				SquashDown:                   "s",
-				RenameCommit:                 "r",
-				RenameCommitWithEditor:       "R",
-				ViewResetOptions:             "g",
-				MarkCommitAsFixup:            "f",
-				CreateFixupCommit:            "F",
-				SquashAboveCommits:           "S",
-				MoveDownCommit:               "<c-j>",
-				MoveUpCommit:                 "<c-k>",
-				AmendToCommit:                "A",
-				PickCommit:                   "p",
-				RevertCommit:                 "t",
-				CherryPickCopy:               "c",
-				CherryPickCopyRange:          "C",
-				PasteCommits:                 "v",
-				TagCommit:                    "T",
-				CheckoutCommit:               "<space>",
-				ResetCherryPick:              "<c-R>",
-				CopyCommitMessageToClipboard: "<c-y>",
-				OpenLogMenu:                  "<c-l>",
-				OpenInBrowser:                "o",
-				ViewBisectOptions:            "b",
+				SquashDown:                     "s",
+				RenameCommit:                   "r",
+				RenameCommitWithEditor:         "R",
+				ViewResetOptions:               "g",
+				MarkCommitAsFixup:              "f",
+				CreateFixupCommit:              "F",
+				SquashAboveCommits:             "S",
+				MoveDownCommit:                 "<c-j>",
+				MoveUpCommit:                   "<c-k>",
+				AmendToCommit:                  "A",
+				PickCommit:                     "p",
+				RevertCommit:                   "t",
+				CherryPickCopy:                 "c",
+				CherryPickCopyRange:            "C",
+				PasteCommits:                   "v",
+				TagCommit:                      "T",
+				CheckoutCommit:                 "<space>",
+				ResetCherryPick:                "<c-R>",
+				CopyCommitAttributeToClipboard: "y",
```

Comment:
just confirming that this is an intentional change i.e. going from '<c-y>' to 'y'

Reply:
Going from `<c-y>` to `y`, you mean? Yes, just like `yank` in `vim`.

Reply:
all good :)

Reply:
Hopefully I won't hit `z` by accident when I forget which keyboard layout is active... :)

Reply:
funny you mention that because I recently added a confirmation behind the 'z' undo key!

Reply:
_Crisis averted!_